bus/bus.go

```diff
@@ -615,9 +615,9 @@ func NewServer(b *Bus) http.Handler {
 		"GET    /contractsets/:name/contracts": b.contractSetContractsHandler,
 		"PUT    /contractsets/:name":           b.contractSetsNameHandlerPUT,
 
-		"GET    /objects/*key":               b.objectsKeyHandlerGET,
-		"PUT    /objects/*key":               b.objectsKeyHandlerPUT,
-		"DELETE /objects/*key":               b.objectsKeyHandlerDELETE,
+		"GET    /objects/store/*key":         b.objectsStoreKeyHandlerGET,
```

Comment:
edit: I just found out Chris kept `/objects/*key` and renamed `objects/migration` to `/migration` (in #38). I considered that as well but figured all store endpoints should live under the same prefix. I don't mind changing it though